package.json

```diff
@@ -115,7 +116,7 @@
     "@googlemaps/jest-mocks": "^2.7.5",
     "@juggle/resize-observer": "^3.3.1",
     "@rollup/plugin-inject": "^4.0.4",
-    "@sage-bionetworks/esbuild-plugin-svgr": "^0.0.4",
+    "esbuild-plugin-svgr": "^1.0.1",
```

Comment:
I had created a PR from our fork that got merged some time ago, so let's go back to the upstream version